apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -565,7 +595,8 @@ function initializeBlocklyWrapper(blocklyInstance) {
       svg.appendChild(workspace.createDom());
       Blockly.cdoUtils.loadBlocksToWorkspace(
         workspace,
-        Blockly.Xml.domToText(xml)
+        Blockly.Xml.domToText(xml),
+        loadHiddenDefinitions
```

Comment:
@mikeharv I agree! Here are some things I've seen address this in the past:
1.  Named parameters are nice for this in other languages, but not relevant in Javascript. 
2. Comments.
3. A named constant to improve readability (as you suggested).
4. Adding an options parameter, so you'd pass `{loadHiddenDefinitions: false}`.
5. Creating wrapper functions with more descriptive names that encapsulates multiple versions of the same behavior.

In this case, I like #3, especially because #4 (my default approach) could get confused with `workspace.options`.

Reply:
using a constant instead!